Assets/_Scripts/Managers/GameManager.cs

```diff
@@ -116,12 +107,13 @@ public void RestartGame()
     {
         _score = 0;
         Lives = 2;
+
         CanvasManager.Instance.UpdateLifeImage(Lives);
-        // will also need to restart the enemy script. 
+        EnemyManager.Instance.Restart();

```

Comment:
Hi There
See you implemented the Game manager enemy reset. 
I will move the card forward.
https://trello.com/c/BkujeIWn/98-gamemanager-reset-enemies-or-game-reset